conans/client/tools/system_pm.py

```diff
@@ -126,6 +126,41 @@ def install(self, packages, update=True, force=False, arch_names=None):
             self.update()
         self._install_any(packages)
 
+    def install_packages(self, packages, update=True, force=False, arch_names=None):
+        """ Get the system package tool install command and install all packages.
+
+        :param packages: String with all packages to be installed e.g. "libusb-dev libfoobar-dev"
+        :param update: Run update command before to install
+        :param force: Force installing all packages
+        :param arch_names: Package suffix/prefix name used by installer tool e.g. {"x86_64": "amd64"}
+        :return: None
+        """
+        packages = [packages] if isinstance(packages, str) else list(packages)
```

Comment:
Wouldn't it make sense to rely on the existing implementation? (we could throw warnings controlled by an undocumented argument). This would also automatically allow the syntax that @jgsogo was proposing for
```python
spt.install([("pkg-variant1", "pkg-variant2"), "otherpkg", "thirdpkg"])
```

That would also probably help with error messages: if some package is not installed now, the ``verify`` mode will raise an error, but it will not tell you which package is missing.

Reply:
I think it would be enough with:

```python
def install_packages(self,...):
   packages = [packages] if isinstance(packages, str) else list(packages)
   for pkg in packages:
      self.install(pkg, update=update, force=force, arch_names=arch_names)
``` 

and it will work with:
```python
spt.install_packages("pkg-variant1")  # (1) String input
spt.install_packages(["pkg-variant1", "otherpkg", "thirdpkg"])  # (2) List of strings: all must be installed
spt.install_packages([("pkg-variant1", "pkg-variant2"), "otherpkg", "thirdpkg"])  # (3) List with alternatives
```

The old method cannot implement (2) without an additional argument to disambiguate if it has to install _all_ packages or _any_ of them.

Reply:
Agree.

Reply:
> Wouldn't it make sense to rely on the existing implementation? (we could throw warnings controlled by an undocumented argument). This would also automatically allow the syntax that @jgsogo was proposing for

I see a breaking behavior, for now, if we pass a list of packages, only one is installed. By passing a new argument to execute a different behavior looks a prone error for me, that's why a preferred a new method.

> I think it would be enough with:
I want to avoid multiple executions of the same command, that's why I "duplicated" the implementation. Any system package manager that we support, can install more than one package by a single command line. I don't think is a good idea re-run for each package name.